config/routes.rb

```diff
@@ -9,6 +9,9 @@
   # get '/post/:user_id/posts/:id', to: 'post#show', as: 'user_post'
 
   resources :users, only: %i[index show] do
-    resources :posts, only: %i[index show]
+    resources :posts, only: %i[index show new create] do
+      resources :comment, only: %i[new index create]
+      resources :like, only: %i[create]
+    end
   end
```

Comment:
- [ **OPTIONAL** ] **IMPORTANT:** I think it would be better if you add a `root path` for your app as right now no one has been implemented yet. I suggest you use the `"users/index"` page for your root path i.e.:
```ruby
  ....
  root "users/index"
  ...
```